Marlin/example_configurations/K8200/language.h

```diff
@@ -0,0 +1,160 @@
+#ifndef LANGUAGE_H
```

Comment:
@thinkyhead Thanks for your advice.

I was thinking about this way, too. But I have been a bit reserved about it, because the printers are nearly the same.

I think you are right and so I will add a new board <BOARD_K8200>. This will make #1383 you merged before obsolete.

This will be a good exercise for me. :-)

URL: I haven't found any repository by them searched GitHub, too. They provide only a ZIP-File on their Website. Bad style. :-(

Should I close this PR and submit a completely new one, or can i update this PR with the changes? Sorry but I am quite new to git and GitHub.